pandas/util/testing.py

```diff
@@ -669,6 +647,19 @@ def assert_numpy_array_equivalent(np_array, assert_equal, strict_nan=False, err_
         err_msg = '{0} is not equivalent to {1}.'.format(np_array, assert_equal)
     raise AssertionError(err_msg)
 
+def assert_numpy_array_equivalent(np_array, assert_equal,
```

Comment:
I don't think we need to deprecate this. Just remove it. These are internal testing routines that we don't want people to use.